src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -505,18 +505,34 @@ private void startFailsafeOperationThread() {
 
   @Override
   public void start(String clientPublicName) throws IOException {
-
     // Construct a single redis client to be used throughout the entire backplane.
     // We wish to avoid various synchronous and error handling issues that could occur when using
     // multiple clients.
     client = new RedisClient(jedisClusterFactory.get());
 
+    this.prequeue = createPrequeue(config);
+    this.operationQueue = createOperationQueue(config);
+
+    if (config.getSubscribeToBackplane()) {
+      startSubscriptionThread();
+    }
+    if (config.getRunFailsafeOperation()) {
+      startFailsafeOperationThread();
+    }
+
+    // Record client start time
+    client.call(
+        jedis -> jedis.set("startTime/" + clientPublicName, Long.toString(new Date().getTime())));
```

Comment:
I see that this was just moved, but it really shouldn't be set without an expiration, and the timestamp acquisition is pretty awkward - why not just make it human readable?

Reply:
maybe the format is best suited for bf-admin? let me see about changing. @80degreeswest is it safe to expire this key?